src/webots/app/WbApplication.hpp

```diff
@@ -48,6 +48,7 @@ class WbApplication : public QObject {
   // worldName must be absolute or specified with respect to WEBOTS_HOME
   // return true on success, false otherwise
   bool loadWorld(QString worldName, bool reloading);
+  bool isValidWorldFile(const QString &worldName);
```

Comment:
The name of this function is a bit misleading as we are not checking the validity of the content of the world file as it suggests, but rather the world file name: does it ends with `.wbt` and does it correspond to a readable file. So I would rename it:
```suggestion
  bool isValidWorldFileName(const QString &worldName);
```